doc/next/_sources/campaign-budgets.rst.txt

```diff
@@ -0,0 +1,71 @@
+.. _campaign_budgets_overview:
+
+Campaign Budgets
+================
+
+Budgets are used to limit the spending for an individual campaign. Your actual spent for a campaign may be lower than the limits you set, depending on a lot of factors; however, it can never be higher.
+Campaign budgets **must be defined**, even if they are intended to be set to *unlimited*. Absence of campaign budgets means that the ads will not be live.
+When ads in this campaign receive clicks, all budgets of this campaign will be checked for remaining balance. In case balance
+runs out on any one of the configured budgets, all ads in this campaign will be taken offline until the budget(s) balance is reset or budgets are increased. The campaign will then be in status *BUDGET_REACHED*.
+
+Budget types
+------------
+
+.. _campaign-budget-types:
+
+
+.. index:: pair: Campaign Budget; Daily
+.. _campaign_budget_daily:
+
+DAILY
+"""""
+
+This is the maximum the campaign can spend per day. Resets every day, close to midnight.
+If no daily budget is set, and you **add** a daily budget after the campaign has already been active, spent for this day will already be taken into account.
+
+.. index:: pair: Campaign Budget; Monthly
+.. _campaign_budget_monthly:
+
+MONTHLY
```

Comment:
Are we going to communicate this is currently not leveraged?

Reply:
but it is (for API partners) and it works, just console doesn't use it/show it. (impending CS questions?)

Reply:
Yhea, shown sorry. We will get CS questions, but that should be fine.